CloverSDK/Utils/CLVServerEnvironment.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ServerEnvironment.swift
+//  CloverDashboard
+//
+//  Created by Yusuf on 12/17/15.
+//  Copyright © 2015 Clover Network, Inc. All rights reserved.
+//
+
+public enum CLVServerEnvironment: String {
+  case US = "https://www.clover.com"
+  case EU = "https://www.eu.clover.com"
+  case DEV1 = "https://dev1.dev.clover.com"
+  case STG1 = "https://stg1.dev.clover.com"
```

Comment:
Since this will be a public SDK, do we want to publicize our internal environments?

I like the convenience of having environments wrapped by enum, but can we also have an init() that accepts a String domain?  That way we can hide the internal enums and also enable other STG and DEV environments should we ever need to connect via iOS SDK.